### DISCUSSION TITLE
What fee should we use when sending xch to another Wallet?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#4176] [Q&A]
What should we use in fee field of the wallet when sending coins? How is this calculated? What is the % needed? This needs some clarification.

**Comment by VictorLaszio** [comment]
if sending to a your wallet on an exchange like gate.io use 0%

**Reply by garyhkirk** [reply]
Fees are only needed when transfer rates are above the block capacity, at the moment 0 fee is fine. If congestion happens, fees are used to get you further up the queue. All transactions are currently fitting in the next transaction block (only time it didn't was when transactions launched and that resolved itself quickly)

**Comment by lvcivs** [comment]
0% is fine for now. A fee will give your transaction a priority, which is only needed when there are lots of transactions going on at the same time and the network has to choose which ones to process first.

**Reply by shhhmel** [reply]
I've got 2.0000000013 solo.
0.250000000013 transferred o wallet
to get 1.75 i need to click Claim rewards ad it asks for fee
Would 0.000000000013 fee be enough?

Afraid if I give 0 fee 1.75 would stuck somewhere.
